autocontingency.lic

```diff
@@ -3,9 +3,6 @@
 =end
 custom_require.call(%w(common common-arcana events spellmonitor drinfomon common-travel))
```

Comment:
My memory is fuzzy, did we make a change to `dependency` such that we no longer need to `custom_require(... drinfomon ...)` or is that still necessary?

Reply:
In theory, dependency always starts drinfomon as script 0.1 (dependency itself being script zero) in practice, someone can still kill drinfomon themselves. Calling it here means if for some weird reason drinfomon isn't already running like it should be, it gets called again. I think it's harmless at worst, and at best it'll start drinfomon in edge cases.